src/components/text/Text.constans.tsx

```diff
@@ -0,0 +1,19 @@
+import { theme } from "../../theme/theme.ts";
+
+export const StylesMap = {
```

Comment:
the object it self is ok, but I'd adjust the name to be more specific about what kind of styles this map is about. 
This is a detail but if you'd have to make 10 such maps for different components and each one would be exported you might find it troublesome to know which export comes from which file. Making the name specific solves the problem.